pom.xml

```diff
@@ -263,7 +263,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-project</artifactId>
-      <version>1.4.1</version>
+      <version>1.4</version>
```

Comment:
you just can't run mvn hpi:run with 1.4.1 - make it hard to test intermediate commits :P